stream/README.md

```diff
@@ -0,0 +1,10 @@
+# commonware-stream
+
+[![Crates.io](https://img.shields.io/crates/v/commonware-stream.svg)](https://crates.io/crates/commonware-stream)
+[![Docs.rs](https://docs.rs/commonware-stream/badge.svg)](https://docs.rs/commonware-stream)
+
+Connect to peers using cryptographically-authenticated connections.
```

Comment:
nit: for now just put placeholder/tbd here (and the places where this string exists)...I think we'll want to put a bit more time into this description.